CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 
 ### Added
 
+* `OpenSslDialer` type (#1).
```

Comment:
I think it's better to have a short useful description like "`OpenSslDialer` type to use SSL transport for `tarantool/go-tarantool/v2` connection".